### ISSUE TITLE
Authenticator page fails to load after browser is launched - Windows

### ISSUE THREAD

**Issue by calumcraig** [maidsafe/safe_browser#719]

**Comment by calumcraig** [comment]
![image](https://user-images.githubusercontent.com/44327924/57066515-4b0c5500-6cc4-11e9-952c-23ab37ae1294.png)

**Comment by calumcraig** [comment]
Console output:
```C:\Users\Calum QA\Downloads\safe-browser-v0.13.0-win-x64\safe-browser-v0.13.0-win-x64>[background 10:22:32.314] ΓÇ║ Getting extension remoteCall Apis
[background 10:22:32.332] ΓÇ║ Registering SAFE Network Protocols
[background 10:22:32.332] ΓÇ║ safe Registering
[background 10:22:32.340] ΓÇ║ Registering safe-auth scheme
[background 10:22:32.340] ΓÇ║ Auth lib location loading:  C:\Users\Calum QA\Downloads\safe-browser-v0.13.0-win-x64\safe-browser-v0.13.0-win-x64\resources\extensions\safe\dist\prod\safe_authenticator.dll
[background 10:22:32.358] ΓÇ║ Peruse internal server listening on port 51570!
[background 10:22:32.358] ΓÇ║ Auth application info { customExecPath:
   [ 'C:\\Users\\Calum QA\\Downloads\\safe-browser-v0.13.0-win-x64\\safe-browser-v0.13.0-win-x64\\SAFE Browser.exe' ],
  icon: 'iconPath',
  id: 'net.maidsafe.app.browser.authenticator',
  name: 'SAFE Browser Authenticator',
  scope: null,
  vendor: 'Maidsafe.net Ltd' }
[background 10:22:32.583] ΓÇ║ Checking urlIsAllowedBySafe file:///C:/Users/Calum%20QA/Downloads/safe-browser-v0.13.0-win-x64/safe-browser-v0.13.0-win-x64/resources/app.asar/app/app.html
[background 10:22:32.583] ΓÇ║ Allowing url file:///C:/Users/Calum%20QA/Downloads/safe-browser-v0.13.0-win-x64/safe-browser-v0.13.0-win-x64/resources/app.asar/app/app.html
[background 10:22:32.731] ΓÇ║ Checking urlIsAllowedBySafe file:///C:/Users/Calum%20QA/Downloads/safe-browser-v0.13.0-win-x64/safe-browser-v0.13.0-win-x64/resources/nessie-styles.css
[background 10:22:32.731] ΓÇ║ Allowing url file:///C:/Users/Calum%20QA/Downloads/safe-browser-v0.13.0-win-x64/safe-browser-v0.13.0-win-x64/resources/nessie-styles.css
[background 10:22:32.745] ΓÇ║ Checking urlIsAllowedBySafe file:///C:/Users/Calum%20QA/Downloads/safe-browser-v0.13.0-win-x64/safe-browser-v0.13.0-win-x64/resources/app.asar/app/dist/style.css
[background 10:22:32.745] ΓÇ║ Allowing url file:///C:/Users/Calum%20QA/Downloads/safe-browser-v0.13.0-win-x64/safe-browser-v0.13.0-win-x64/resources/app.asar/app/dist/style.css
[background 10:22:32.751] ΓÇ║ Checking urlIsAllowedBySafe file:///C:/Users/Calum%20QA/Downloads/safe-browser-v0.13.0-win-x64/safe-browser-v0.13.0-win-x64/resources/app.asar/app/dist/renderer.prod.js
[background 10:22:32.753] ΓÇ║ Allowing url file:///C:/Users/Calum%20QA/Downloads/safe-browser-v0.13.0-win-x64/safe-browser-v0.13.0-win-x64/resources/app.asar/app/dist/renderer.prod.js
[window 2 10:22:33.602] ΓÇ║ Extending Component via the extensions Api
[window 2 10:22:33.613] ΓÇ║ Extending Pe via the extensions Api
[window 2 10:22:33.613] ΓÇ║ Extending Be via the extensions Api
[window 2 10:22:34.423] ΓÇ║ Extending oa via the extensions Api
[window 2 10:22:34.432] ΓÇ║ Starting render process
[window 2 10:22:34.452] ΓÇ║ Getting extension browser actions
[window 2 10:22:34.452] ΓÇ║ Wrapping browser
[background 10:22:34.563] ΓÇ║ Authenticator lib status:  true
[background 10:22:34.563] ΓÇ║ Initialising Safe Browser App with options: { enableExperimentalApis: false, forceUseMock: false }
[background 10:22:34.563] ΓÇ║ Initing anon connection with options: { enableExperimentalApis: false,
  forceUseMock: false,
  joinSchemes: [ 'safe' ],
  libPath: 'C:\\Users\\Calum QA\\Downloads\\safe-browser-v0.13.0-win-x64\\safe-browser-v0.13.0-win-x64\\resources\\app.asar.unpacked\\node_modules\\@maidsafe\\safe-node-app\\src\\native',
  registerScheme: true }
[background 10:22:34.753] ΓÇ║ Initialising Safe Browser App with options: { enableExperimentalApis: false, forceUseMock: false }
[background 10:22:34.753] ΓÇ║ Initing anon connection with options: { enableExperimentalApis: false,
  forceUseMock: false,
  joinSchemes: [ 'safe' ],
  libPath: 'C:\\Users\\Calum QA\\Downloads\\safe-browser-v0.13.0-win-x64\\safe-browser-v0.13.0-win-x64\\resources\\app.asar.unpacked\\node_modules\\@maidsafe\\safe-node-app\\src\\native',
  registerScheme: true }
[background 10:22:34.793] ΓÇ║ Decoding an auth req object { id: 325841607,
  uri: 'safe-auth:bAAAAAAGH6JVRGAQAAAABSAAAAAAAAAAANZSXILTNMFUWI43BMZSS443BMZSS2YTSN53XGZLSAA' }
[background 10:22:34.793] ΓÇ║ IPC.js enqueue authQ req...
[background 10:22:34.793] ΓÇ║ IPC.js adding req
[background 10:22:34.793] ΓÇ║ IPC.js pushing req
[background 10:22:34.793] ΓÇ║ Authenticator.js decoding request safe-auth:bAAAAAAGH6JVRGAQAAAABSAAAAAAAAAAANZSXILTNMFUWI43BMZSS443BMZSS2YTSN53XGZLSAA
[window 2 10:22:35.162] ΓÇ║ Wrapping Address bar buttons LHS
[window 2 10:22:35.205] ΓÇ║ Wrapping Address bar input
[window 2 10:22:35.205] ΓÇ║ Wrapping Address bar buttons RHS
[background 10:22:35.916] ΓÇ║ IPC.js: another response being parsed.: { error: null,
  id: 325841607,
  res: 'safe-bmv0lm1hawrzywzllnnhzmutynjvd3nlcg:bAEAAAAGH6JVRGAQAAAAAAAAAAAMQAAAAAAAAAAAQAAAAAAAAAAADCNZYFY3DELRXGYXDQORVGQ4DGEYAAAAAAAAAAAYTGOBOGY4C4MJYGUXDEMJYHI2TIOBTCIAAAAAAAAAAAMJTHAXDMOBOGE4DCLRVG45DKNBYGMJAAAAAAAAAAABRGM4C4NRYFYYTQMJOGYYDUNJUHAZREAAAAAAAAAAAGEZTQLRWHAXDCOBRFY4DMORVGQ4DGEQAAAAAAAAAAAYTGOBOGY4C4MJYGEXDQNZ2GU2DQMYTAAAAAAAAAAADCMZYFY3DQLRRHAYS4MJWHA5DKNBYGMJQAAAAAAAAAABRGM4C4NRYFYYTQMJOGE3TMORVGQ4DGEYAAAAAAAAAAAYTGOBOGY4C4MJYGEXDCNZZHI2TIOBTCMAAAAAAAAAAAMJTHAXDMOBOGE4DCLRRHAYDUNJUHAZRGAAAAAAAAAAAGEZTQLRWHAXDCOBRFYYTQMR2GU2DQMYTAAAAAAAAAAADCMZYFY3DQLRRHAYS4MRUGI5DKNBYGMJQAAAAAAAAAABRGM4C4NRYFYYTQMJOGI2DGORVGQ4DGEYAAAAAAAAAAAYTGOBOGY4C4MJYGEXDENBZHI2TIOBTCIAAAAAAAAAAAMJTHAXDMOBOGE4DSLRRGQ5DKNBYGMJAAAAAAAAAAABRGM4C4NRYFYYTQOJOGE2TUNJUHAZREAAAAAAAAAAAGEZTQLRWHAXDCOBZFYYTOORVGQ4DGEQAAAAAAAAAAAYTGOBOGY4C4MJYHEXDCOB2GU2DQMYSAAAAAAAAAAADCMZYFY3DQLRRHA4S4MJZHI2TIOBTCIAAAAAAAAAAAMJTHAXDMOBOGE4DSLRTGE5DKNBYGMJAAAAAAAAAAABRGM4C4NRYFYYTQOJOGM2DUNJUHAZREAAAAAAAAAAAGEZTQLRWHAXDCOBZFYZTMORVGQ4DGEQAAAAAAAAAAAYTGOBOGY4C4MJYHEXDGOB2GU2DQMYSAAAAAAAAAAADCMZYFY3DQLRRHA4S4MZZHI2TIOBTCEAAAAAAAAAAANBWFYYTAMJOGUXDCNZZHI2TIOBTAFVRKAAAAAAAAAIHAAAAAAAAAAAGC3DQNBQV6MQAAA',
  type: 'DESKTOP',
  uri: 'safe-auth:bAAAAAAGH6JVRGAQAAAABSAAAAAAAAAAANZSXILTNMFUWI43BMZSS443BMZSS2YTSN53XGZLSAA' }
[background 10:22:35.923] ΓÇ║ Decoding an auth req object { id: 1982933960,
  uri: 'safe-auth:bAAAAAAGIFMYXMAQAAAABSAAAAAAAAAAANZSXILTNMFUWI43BMZSS443BMZSS2YTSN53XGZLSAA' }
[background 10:22:35.923] ΓÇ║ IPC.js enqueue authQ req...
[background 10:22:35.923] ΓÇ║ IPC.js adding req
[background 10:22:35.933] ΓÇ║ IPC.js pushing req
[background 10:22:35.933] ΓÇ║ Authenticator.js decoding request safe-auth:bAAAAAAGIFMYXMAQAAAABSAAAAAAAAAAANZSXILTNMFUWI43BMZSS443BMZSS2YTSN53XGZLSAA
[background 10:22:35.933] ΓÇ║ IPC.js: another response being parsed.: { error: null,
  id: 1982933960,
  res: 'safe-bmv0lm1hawrzywzllnnhzmutynjvd3nlcg:bAEAAAAGIFMYXMAQAAAAAAAAAAAMQAAAAAAAAAAAQAAAAAAAAAAADCNZYFY3DELRXGYXDQORVGQ4DGEYAAAAAAAAAAAYTGOBOGY4C4MJYGUXDEMJYHI2TIOBTCIAAAAAAAAAAAMJTHAXDMOBOGE4DCLRVG45DKNBYGMJAAAAAAAAAAABRGM4C4NRYFYYTQMJOGYYDUNJUHAZREAAAAAAAAAAAGEZTQLRWHAXDCOBRFY4DMORVGQ4DGEQAAAAAAAAAAAYTGOBOGY4C4MJYGEXDQNZ2GU2DQMYTAAAAAAAAAAADCMZYFY3DQLRRHAYS4MJWHA5DKNBYGMJQAAAAAAAAAABRGM4C4NRYFYYTQMJOGE3TMORVGQ4DGEYAAAAAAAAAAAYTGOBOGY4C4MJYGEXDCNZZHI2TIOBTCMAAAAAAAAAAAMJTHAXDMOBOGE4DCLRRHAYDUNJUHAZRGAAAAAAAAAAAGEZTQLRWHAXDCOBRFYYTQMR2GU2DQMYTAAAAAAAAAAADCMZYFY3DQLRRHAYS4MRUGI5DKNBYGMJQAAAAAAAAAABRGM4C4NRYFYYTQMJOGI2DGORVGQ4DGEYAAAAAAAAAAAYTGOBOGY4C4MJYGEXDENBZHI2TIOBTCIAAAAAAAAAAAMJTHAXDMOBOGE4DSLRRGQ5DKNBYGMJAAAAAAAAAAABRGM4C4NRYFYYTQOJOGE2TUNJUHAZREAAAAAAAAAAAGEZTQLRWHAXDCOBZFYYTOORVGQ4DGEQAAAAAAAAAAAYTGOBOGY4C4MJYHEXDCOB2GU2DQMYSAAAAAAAAAAADCMZYFY3DQLRRHA4S4MJZHI2TIOBTCIAAAAAAAAAAAMJTHAXDMOBOGE4DSLRTGE5DKNBYGMJAAAAAAAAAAABRGM4C4NRYFYYTQOJOGM2DUNJUHAZREAAAAAAAAAAAGEZTQLRWHAXDCOBZFYZTMORVGQ4DGEQAAAAAAAAAAAYTGOBOGY4C4MJYHEXDGOB2GU2DQMYSAAAAAAAAAAADCMZYFY3DQLRRHA4S4MZZHI2TIOBTCEAAAAAAAAAAANBWFYYTAMJOGUXDCNZZHI2TIOBTAFVRKAAAAAAAAAIHAAAAAAAAAAAGC3DQNBQV6MQAAA',
  type: 'DESKTOP',
  uri: 'safe-auth:bAAAAAAGIFMYXMAQAAAABSAAAAAAAAAAANZSXILTNMFUWI43BMZSS443BMZSS2YTSN53XGZLSAA' }
[background 10:22:39.813] ΓÇ║ Authing from a store-passed response. 1556788959813 safe-bmv0lm1hawrzywzllnnhzmutynjvd3nlcg:bAEAAAAGH6JVRGAQAAAAAAAAAAAMQAAAAAAAAAAAQAAAAAAAAAAADCNZYFY3DELRXGYXDQORVGQ4DGEYAAAAAAAAAAAYTGOBOGY4C4MJYGUXDEMJYHI2TIOBTCIAAAAAAAAAAAMJTHAXDMOBOGE4DCLRVG45DKNBYGMJAAAAAAAAAAABRGM4C4NRYFYYTQMJOGYYDUNJUHAZREAAAAAAAAAAAGEZTQLRWHAXDCOBRFY4DMORVGQ4DGEQAAAAAAAAAAAYTGOBOGY4C4MJYGEXDQNZ2GU2DQMYTAAAAAAAAAAADCMZYFY3DQLRRHAYS4MJWHA5DKNBYGMJQAAAAAAAAAABRGM4C4NRYFYYTQMJOGE3TMORVGQ4DGEYAAAAAAAAAAAYTGOBOGY4C4MJYGEXDCNZZHI2TIOBTCMAAAAAAAAAAAMJTHAXDMOBOGE4DCLRRHAYDUNJUHAZRGAAAAAAAAAAAGEZTQLRWHAXDCOBRFYYTQMR2GU2DQMYTAAAAAAAAAAADCMZYFY3DQLRRHAYS4MRUGI5DKNBYGMJQAAAAAAAAAABRGM4C4NRYFYYTQMJOGI2DGORVGQ4DGEYAAAAAAAAAAAYTGOBOGY4C4MJYGEXDENBZHI2TIOBTCIAAAAAAAAAAAMJTHAXDMOBOGE4DSLRRGQ5DKNBYGMJAAAAAAAAAAABRGM4C4NRYFYYTQOJOGE2TUNJUHAZREAAAAAAAAAAAGEZTQLRWHAXDCOBZFYYTOORVGQ4DGEQAAAAAAAAAAAYTGOBOGY4C4MJYHEXDCOB2GU2DQMYSAAAAAAAAAAADCMZYFY3DQLRRHA4S4MJZHI2TIOBTCIAAAAAAAAAAAMJTHAXDMOBOGE4DSLRTGE5DKNBYGMJAAAAAAAAAAABRGM4C4NRYFYYTQOJOGM2DUNJUHAZREAAAAAAAAAAAGEZTQLRWHAXDCOBZFYZTMORVGQ4DGEQAAAAAAAAAAAYTGOBOGY4C4MJYHEXDGOB2GU2DQMYSAAAAAAAAAAADCMZYFY3DQLRRHA4S4MZZHI2TIOBTCEAAAAAAAAAAANBWFYYTAMJOGUXDCNZZHI2TIOBTAFVRKAAAAAAAAAIHAAAAAAAAAAAGC3DQNBQV6MQAAA
[window 2 10:22:40.902] ΓÇ║ Setup up SAFE Dom API via @maidsafe/safe-node-app
[window 2 10:22:40.962] ΓÇ║ Webview: loading url: safe-auth://home/
[window 2 10:22:41.012] ΓÇ║ Tab did stop loading
[window 2 10:22:41.012] ΓÇ║ Tab did finish loading
[window 2 10:22:41.012] ΓÇ║ Tab did stop loading
[window 2 10:22:41.024] ΓÇ║ webview started loading
[background 10:22:42.313] ΓÇ║ onNetworkStateChange
[background 10:22:42.313] ΓÇ║ previousState:  null
[background 10:22:42.343] ΓÇ║ Checking urlIsAllowedBySafe safe-auth://home/
[background 10:22:42.343] ΓÇ║ Allowing url safe-auth://home/
[background 10:22:42.343] ΓÇ║ Procotol:: safe-auth:// url being parsed: safe-auth://home/
[background 10:22:42.343] ΓÇ║ Checking urlIsAllowedBySafe http://localhost:51570/auth/safe-auth://home/
[background 10:22:42.353] ΓÇ║ Allowing url http://localhost:51570/auth/safe-auth://home/
[window 2 10:22:43.443] ΓÇ║ webview did navigate
[window 2 10:22:43.494] ΓÇ║ Tab: did receive updated props
[window 2 10:22:43.502] ΓÇ║ Tab: did receive updated props
[window 2 10:22:43.502] ΓÇ║ Webview focussed: Triggering click event on browser window
[window 2 10:22:43.513] ΓÇ║ Tab: did receive updated props
[window 2 10:22:45.042] ΓÇ║ Webview: did navigate in page safe-auth://home/#/ safe-auth://home/
[window 2 10:22:45.052] ΓÇ║ Setup up SAFE Dom API via @maidsafe/safe-node-app
[background 10:22:45.063] ΓÇ║ Checking urlIsAllowedBySafe safe-auth://home/bundle.js
[window 2 10:22:45.083] ΓÇ║ Webview: page title updated
[background 10:22:45.083] ΓÇ║ Allowing url safe-auth://home/bundle.js
[background 10:22:45.093] ΓÇ║ Procotol:: safe-auth:// url being parsed: safe-auth://home/bundle.js
[background 10:22:45.113] ΓÇ║ Checking urlIsAllowedBySafe http://localhost:51570/auth/safe-auth://home/bundle.js
[background 10:22:45.113] ΓÇ║ Allowing url http://localhost:51570/auth/safe-auth://home/bundle.js```

**Comment by calumcraig** [comment]
log file:
```T 10:22:39.9743103 [C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\crust-0.31.0\src\main\service.rs:562] Network name: Some("alpha_2")
T 10:22:41.1847977 [C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\crust-0.31.0\src\main\service.rs:78] Event loop started
T 10:22:41.1922376 [C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\crust-0.31.0\src\main\config_refresher.rs:36] Entered state ConfigRefresher
I 10:22:42.2519620 [C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\crust-0.31.0\src\main\bootstrap\mod.rs:197] Failed to Bootstrap with 138.68.181.249:5483: (ClientNotWhitelisted) Our Client is not whitelisted
T 10:22:42.2555701 [C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\crust-0.31.0\src\main\active_connection.rs:55] Entered state ActiveConnection: PublicId(name: 4be500..) -> PublicId(name: e17ace..)
T 10:22:42.2555701 [C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\crust-0.31.0\src\main\active_connection.rs:100] Connection Map inserted: PublicId(name: e17ace..) -> Some(ConnectionId { active_connection: Some(Token(8)), currently_handshaking: 0 })
D 10:22:42.2555701 [<unknown>:249]                Bootstrapping(4be500..) Received BootstrapConnect from e17ace...
D 10:22:42.2555701 [<unknown>:315]                Bootstrapping(4be500..) Sending BootstrapRequest to e17ace...
D 10:22:42.3132791 [<unknown>:84]                 Client(4be500..) State changed to client.```

**Comment by manavbp** [comment]
This issue was fixed by #751

**Comment by safesurfer** [comment]
QA verified
